frontend/components/Dashboard/Profile/ProfileCard.tsx

```diff
@@ -31,20 +31,34 @@ const ProfileCard: React.FC<Props> = ({ currentUser }) => {
 
   const name = currentUser.name || currentUser.handle
   const showSeparator = user.facebook || user.instagram || user.youtube || user.website
+  const profileImage = currentUser.profileImage
+  const speaks = []
```

Comment:
if you want to do it in one line you could do `const speaks = currentUser.langaugesNative.map(({ language }) => language.name)` 😄

Reply:
Oh right! It's funny, I originally started writing it as a Python list comprehension and then was like awww can't do those, and forgot about good ol' `map` haha 😂

Reply:
Eh, everything is blowing up with TS when I do that and I'm exhausted 😅🙈